platform-core-module/src/main/java/com/softicar/platform/core/module/page/service/login/PageServiceLoginDiv.java

```diff
@@ -48,13 +49,15 @@ private class LoginForm extends DomForm {
 		private final LoginButton loginButton;
 		private final UserInput userInput;
 		private final PasswordInput passwordInput;
+		private final MaintenanceElement maintenanceDiv;
```

Comment:
IMO, should be `MaintenanceElement maintenanceElement;` or `MaintenanceDiv maintenanceDiv;`

Reply:
can do in another cleanup revision